docs/application-tutorial/game_of_life.py

```diff
@@ -0,0 +1,551 @@
+import sys
```

Comment:
import order could be consistent

Reply:
what do you mean, sorry? I guess anyway, @umut-sahin said that the CI would launch, so I will be caught by your linters

Reply:
You can run `make conformance` before pushing, it should be formatted correctly :+1:

Reply:
thx done!